spec/features/users/index.html.erb_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe 'User Index', type: :feature do
+  before :each do
+    @user1 = User.create(
+      name: 'Gaby',
+      photo: 'bulma.jpg',
+      bio: 'Systems Engineering Student'
+    )
+    @user2 = User.create(
+      name: 'Leo',
+      photo: 'vegeta.jpg',
+      bio: 'Law Student'
+    )
+    @user1_url = "a[href = '#{user_path(@user1.id)}']"
+    @user2_url = "a[href = '#{user_path(@user2.id)}']"
+    visit users_path
+  end
+  it 'shows the right content' do
+    [@user1, @user2].each do |user|
+      expect(page).to have_content(user.name)
+      expect(page).to have_css("img[src*='#{user.photo}']")
+      expect(page).to have_content("Number of posts: #{user.posts_counter}")
+    end
+  end
+  it "When I click on a user, I am redirected to that user's show page" do
+    find(@user1_url).click
+    expect(page).to have_content(@user1.bio)
+  end
+  it "When I click on a user, I am redirected to that user's show page" do
+    find(@user2_url).click
+    expect(page).to have_content(@user2.bio)
+  end
+end
```

Comment:
- [ ] For some reason, when I run rspec to check whether the specs run or not, I am getting this error. Kindly look into why this is the case.

![image](https://user-images.githubusercontent.com/86318284/216712692-7afc66e8-933f-4074-81ce-f134f13dad13.png)